spec/datadog/tracing/contrib/support/tracer_helpers.rb

```diff
@@ -136,5 +136,16 @@ def use_real_tracer!
       @use_real_tracer = true
       allow(Datadog::Tracing::Tracer).to receive(:new).and_call_original
     end
+
+    # Wraps call to Tracing::Utils::TraceId.to_low_order for better test readability
+    def low_order_trace_id(trace_id)
+      Datadog::Tracing::Utils::TraceId.to_low_order(trace_id)
+    end
+
+    # Wraps call to Tracing::Utils::TraceId.to_high_order and converts to hex
+    # for better test readability
+    def high_order_hex_trace_id(trace_id)
+      Datadog::Tracing::Utils::TraceId.to_high_order(trace_id).to_s(16)
+    end
```

Comment:
I think these helpers are related to tracing, and not specifically contrib, as the 128-bit trace id generation is a part of core tracing.

I suggest moving these helpers to https://github.com/DataDog/dd-trace-rb/blob/master/spec/support/tracer_helpers.rb instead.